Backend/alumni_portal/settings.py

```diff
@@ -143,4 +143,4 @@
 
 TELEGRAM_TOKEN = "6125230376:AAGi7qfothkdpDGwwy7nB9x8VieXwzN9yNQ"
 CHAT_ID = -1001525464247
-EMAIL_PASSWORD = "fkCySytmUKS2t5"
+EMAIL_PASSWORD = ""
```

Comment:
Why ?

Reply:
I deactivated anything to do directly with the university mail server. It hangs
I will use some kinda queue for the messages or try to remove these steps all together

Reply:
okay can you just add a comment about it ? TODO later add a queue ..etc? just for understanding why this is commented